html2text/utils.py

```diff
@@ -244,3 +244,51 @@ def escape_md_section(text, snob=False):
     text = config.RE_MD_DASH_MATCHER.sub(r"\1\\\2", text)
 
     return text
+
+def reformat_table(lines, right_margin):
+    """
+    Given the lines of a table
+    padds the cells and returns the new lines
+    """
+    # find the maximum width of the columns
+    max_width = [len(x.rstrip()) + right_margin for x in lines[0].split('|')]
+    for line in lines:
+        cols = [x.rstrip() for x in line.split('|')]
+        max_width = [max(len(x) + right_margin, old_len) for x, old_len in zip(cols, max_width)]
```

Comment:
No problem, leave it out.
We still don't have strict rules for pep8 or code quality.
We'll be cleaning them with #131